students/Kate_Koehn/Lesson10/database.py

```diff
@@ -0,0 +1,166 @@
+""""
+Program to query products and customers in the rental database.
+
+"""
+import csv
+from pymongo import MongoClient
+from timeit import time
+
+
+def timing(func):
+    def wrap(*args):
+        time1 = time.time()
+        end = func(*args)
+        time2 = time.time()
+        print('{:s} function took {:.3f} ms'.format(func.__name__, (time2-time1)*1000.0))
+        with open("timing_output.txt", "a") as output_file:
+            output_file.write('{:s} function took {:.3f} ms\n'.format(func.__name__, (time2-time1)*1000.0))
+        return end
+    return wrap
+
+
+@timing
+class MongoDBConnection():
+    """MongoDB Connection"""
```

Comment:
Using the timing decorator on this class is kind of weird. You need special decorators for classes, otherwise you are just timing how long it took to instantiate the class, which is not terribly useful. What you would really want to know is how long it took to connect to the database .